### PR TITLE
Bump fiji to latest lifeline version

### PR DESCRIPTION
**Pull request by gregvonkuster**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by gregvonkuster** [comment]
@bgruening any idea why this error now occurs and what I need to change to eliminate it?
```
ERROR: recipes/fiji/20170530/meta.yaml:8: should_not_be_noarch_source: The recipe uses per platform sources and cannot be noarch
```

File: recipes/fiji/20170530/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      md5: a5c9af6c0ced968931faa3280e2394e1   # [osx]
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - openjdk
+  run:
+    - openjdk
```
Comment (bgruening) [inline recipes/fiji/20170530/meta.yaml]: @gregvonkuster I think you can add python as runtime dep here, if jython is a use-case.

File: recipes/fiji/20170530/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  requires:
+    - python
+  files:
+    - hello.py
```
Comment (bgruening) [inline recipes/fiji/20170530/meta.yaml]: This does unfortunately not work with the current bioconda setup.
You can either echo "script" into a file and use that ... or use a `run_test.sh` file.

Comment (gregvonkuster) [reply recipes/fiji/20170530/meta.yaml]: Thanks for the help @bgruening  - I'll take another pass at this with your info.

Comment (gregvonkuster) [reply recipes/fiji/20170530/meta.yaml]: @bgruening it's looking happy now - thanks!  The Fiji package includes all that's needed for Jython, so the Python requirement isn't necessary.

**Comment by bgruening** [comment]
Greg is that no a newer release? https://downloads.imagej.net/fiji/Dresden-2009/

**Comment by gregvonkuster** [comment]
I've always used the latest Fiji lifeline version - https://imagej.net/Fiji/Downloads.  You can always build install recipes for the source code https://imagej.net/Source_code, but when I was looking at this years ago, the lifeline versions were always stable and production ready.  It looks like your Dresden link points to a 2009 version anyway, while the latest Fiji lifeline that I'm installing in this recipe is from 2017.

**Comment by bgruening** [comment]
So true, sorry for the noise!

**Pull request merged**